### PR TITLE
Admin authorization

### PR DESCRIPTION
**Pull request by kpearson**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -11,13 +12,18 @@ def set_cart
...
-
   private
 
   def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    @current_user ||= Admin.find(session[:admin_id]) if session[:admin_id]
+    @current_user
   end
 
+  # def current_admin
```
Comment (bryceholcomb) [inline app/controllers/application_controller.rb]: can we remove the commented out code

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -11,13 +12,18 @@ def set_cart
     @cart = Cart.new(session[:cart])
   end
 
-  before_action :set_cart
-
   private
 
   def current_user
```
Comment (bryceholcomb) [inline app/controllers/application_controller.rb:17]: Glad we were able to make this work

Comment (kpearson) [inline app/controllers/application_controller.rb:17]: Never any doubt...

**Comment by scottalexandra** [comment]
Good Work!

**Pull request merged**